### DISCUSSION TITLE
BSHR Full implementation using Bing Web Search API

### DISCUSSION THREAD

**Discussion by joseph-crowley** [daveshap/BSHR_Loop#10] [Ideas]
Hey Dave, 

I've created a [BSHR repo](https://github.com/joseph-crowley/BSHR) to implement a starting point for a complete project based on [your pseudocode demo](https://github.com/daveshap/BSHR_Loop/blob/main/demo01/bshr_loop.py).

Check it out! Please let me know what you think.

Best,
Joe

**Comment by daveshap** [comment]
This looks really great! Thanks for sharing :)
